app/views/case_contacts/form/_contact_types.html.erb

```diff
@@ -3,7 +3,7 @@
   <%= render(Form::MultipleSelectComponent.new(
     form: form,
     name: :contact_type_ids,
-    options: options.decorate.map { |ct| ct.hash_for_multi_select_with_cases(casa_cases&.pluck(:id)) },
+    options: options.decorate.sort_by { |ct| ct.last_contact_timestamp(casa_cases.pluck(:id)) }.reverse.map { |ct| ct.hash_for_multi_select_with_cases(casa_cases.pluck(:id)) },
```

Comment:
I think the ordering can be done in https://github.com/rubyforgood/casa/blob/ed7b83fde6c7bbf58af382401afe35f7d7d914fc/app/controllers/case_contacts/form_controller.rb#L63

Reply:
I tried to do this originally in the controller but couldn't get it to work. Any suggestions? 

The decorator wasn't working correctly specifically if I remember correctly